dapp-core/src/main/kotlin/jp/co/soramitsu/dapp/main.kt

```diff
@@ -0,0 +1,21 @@
+@file:JvmName("DappMain")
+
+package jp.co.soramitsu.dapp
+
+import jp.co.soramitsu.dapp.service.DappService
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import org.springframework.context.annotation.AnnotationConfigApplicationContext
+import org.springframework.context.annotation.ComponentScan
+
+@ComponentScan("jp.co.soramitsu.dapp")
+class DappApplication
+
+fun main(args: Array<String>) {
+    val context = AnnotationConfigApplicationContext()
+    context.register(DappApplication::class.java)
```

Comment:
No need to call `register` and `refresh`

Reply:
It is. I get `AnnotationConfigApplicationContext@3abfe836 has not been refreshed yet` otherwise

Reply:
Have you tried `AnnotationConfigApplicationContext(DappApplication::class.java)`?

Reply:
I don' think that the issue is solved.